src/App.js

```diff
@@ -1,12 +1,28 @@
+import { useEffect } from "react"
+import { useDispatch } from "react-redux"
 import { Route, Routes } from "react-router-dom";
+import { getUsers } from "./features/auth/authSlice";
+import { getPosts } from "./features/post/postSlice"
+import { Toaster } from 'react-hot-toast';
 import {  AuthRoute, PrivateRoute } from "./components";
 import { Login, Home, Profile, Explore, Bookmark, Mock} from "./Pages"
 
 import "./style.css"
 
 function App() {
+  const dispatch = useDispatch()
+
+  useEffect(()=>{
+     dispatch(getPosts())
+  },[])
+
+  useEffect(()=>{
+    dispatch(getUsers())
+  },[])
+
   return (
    <>
+     <Toaster/>
      <Routes>
        <Route path="/mock" element={<Mock/>} />
        <Route element={<PrivateRoute/>}>
```

Comment:
Just a small suggestion make a new component for all the routes and import that here it will make the app impressive.